### ISSUE TITLE
Add extra flags to Ganache

### ISSUE THREAD

**Issue by apguerrera** [iamdefinitelyahuman/brownie#486]
### Overview
I wanted to add more ETH to the test accounts, and needed to set the -e flag in ganache
I noticed it was cutting out the extra commands, but could be added

### Specification

brownie-config.yaml

```
network:
   networks:
       development:
            test_rpc:
                cmd: ganache-cli
                extra_param: "-e  10000"
```

or 

```
network:
   networks:
       development:
            test_rpc:
                cmd: "ganache-cli -e 10000"
                
```

otherwise less generally
 
```
network:
   networks:
       development:
            test_rpc:
                cmd: ganache-cli
                default_eth: 10000
```

**Comment by iamdefinitelyahuman** [comment]
Internally, network settings are stored at `~/.brownie/network-config.yaml`.  For development networks, all ganache related flags are in the `cmd_settings` field.

Some of the existing fields, and the new field proposed in this issue, are definitely project-dependent. I think the best approach is to expose `cmd_settings` within `brownie-config.yaml`, something like this:

```yaml
networks:
    default: development
    development:
        cmd_settings:
            block_time: 10
            default_balance: "1000 ether"
```

**Comment by matnad** [comment]
closed by #501?
